particles/Experimental/Person_Msg_Demo/PersonMsg.arcs

```diff
@@ -0,0 +1,69 @@
+// Demo: Person-Msg
+
+schema Person
+  name: Text
+  id: Text
+  age: Number [age > 0]
+
+schema Message
+  toID: Text
+  fromID: Text
+  content: Text
+  sentTime: Number [sentTime > 0]
```

Comment:
Please add a comment that currently `sentTime` is 'time since message was sent (in seconds)'.
This will help explain the refinement here (which I thought was a 'technically correct' bug, assuming that no messages were sent before 1970 [epoch]).